src/pudl/output/pudltabl.py

```diff
@@ -199,141 +208,50 @@ def _get_table_from_db(self, table_name: str, resource: Resource) -> pd.DataFram
         return pd.concat(
             [
                 resource.enforce_schema(df)
-                for df in pd.read_sql(table_name, self.pudl_engine, chunksize=100_000)
+                for df in pd.read_sql(
+                    self._select_between_dates(table_name),
+                    self.pudl_engine,
+                    chunksize=100_000,
+                )
             ]
         )
 
-    def pu_eia860(self, update=False):
-        """Pull a dataframe of EIA plant-utility associations.
+    def _select_between_dates(self, table: str) -> sa.sql.expression.Select:
```

Comment:
Mostly this is a name change, but also needed to make it work on `report_date` + `report_year` + tables with no date column.